macros/cross_db_utils/width_bucket.sql

```diff
@@ -0,0 +1,29 @@
+{% macro width_bucket(expr, min_value, max_value, num_buckets) %}
+  {{ adapter_macro('dbt_utils.width_bucket', expr, min_value, max_value, num_buckets) }}
+{% endmacro %}
+
+
+{% macro default__width_bucket(expr, min_value, max_value, num_buckets) -%}
+
+    {%- set bucket_width = '((' ~ max_value ~ ' - ' ~ min_value ~ ')/' ~ num_buckets ~ ')' -%}
+    {%- set to_numeric = '::numeric' -%}
```

Comment:
I don't think this will work on BQ - can you try using the [safe_cast](https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/cross_db_utils/safe_cast.sql) macro here instead? Or just 
```
cast({{ bucket_width }} as numeric)
```

Reply:
Makes sense, changed pushed!